sema/syncfs/service.py

```
@@ -48,8 +49,10 @@ def get_lastmod_by_fname(from_path: Path) -> Dict[str, datetime]:
     """
     return {
         str(p): datetime.fromtimestamp(p.stat().st_mtime, UTC_tz)
-        for p in from_path.glob("**/*")
-        if p.is_file() and is_supported_rdffilepath(p)
+        for p in getMatchingGlobPaths(
+            from_path, onlyFiles=True, makeRelative=False
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)